client/src/components/SchedulesDisplay.tsx

```diff
@@ -115,10 +136,17 @@ export const SchedulesDisplay = ({ course }: SchedulesDisplayProps) => {
                 <span className='font-semibold'>Campus: </span>
                 {block.campus}
               </div>
-              <div className='w-80'>
-                <span className='font-semibold'>Classroom(s): </span>
-                {block.location ? block.location.replace(';', ',') : 'N/A'}
-              </div>
+              <span className='font-semibold'>Classroom(s):</span>
+              <span className='inline-block w-80'>
+                {block.location.split(';').map((location: string, index) => (
+                  <span>
+                    <BlockLocation location={location.trim()} />
+                    {index !== block.location.split(';').length - 1 && (
```

Comment:
Can we take out `block.location.split(';')` so we don't have to call it multiple times?